### ISSUE TITLE
sshfs: command not found

### ISSUE THREAD

**Issue by oxr463** [dustymabe/vagrant-sshfs#110]
Here is are the Vagrantfile contents:

```
Vagrant.configure("2") do |config|
  config.vm.box = "generic/gentoo"
  config.vm.synced_folder ".", "/tmp", type: "sshfs", reverse: false
end
```

Here is the actual output:

```sh
vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'generic/gentoo'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'generic/gentoo' version '3.0.0' is up to date...
==> default: Setting the name of the VM: quickstart_default_1589045623687_57353
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: 
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default: 
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if it's present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
==> default: Mounting SSHFS shared folder...
==> default: Mounting folder via SSHFS: /home/user/tmp => /tmp
==> default: Checking Mount..
==> default: Checking Mount..
==> default: Checking Mount..
==> default: Checking Mount..
==> default: Checking Mount..
==> default: Checking Mount..
==> default: Checking Mount..
Mounting SSHFS shared folder via slave SSHFS mount failed. Please
look at the below STDERR output from the processes that were run.

SSH command:

Warning: Permanently added '[127.0.0.1]:2222' (ECDSA) to the list of known hosts.
sudo: sshfs: command not found


SFTP command:

```

See also: https://github.com/dustymabe/vagrant-sshfs/issues/62

**Comment by oxr463** [comment]
```sh
sudo emerge -q net-fs/sshfs
```

**Comment by localhost8080** [comment]
Found an actual workaround (or two)
The actual fix is this: https://github.com/dustymabe/vagrant-sshfs/pull/111 and it shows the reason: 

Vagrant 2.8.8 introduced explicit handling of CentOS as a guest in the commit hashicorp/vagrant@5104d07

This means that on boot in vagrant >= 2.2.8 , centos is not detected (which means that the epel repo is not installed, which means that fuse-sshfs is not installed)

Workaround is to use the code in the pull request (or if thats out of reach, downgrade vagrant to 2.2.7)

Hopefully the referenced pull request gets merged in soon so we can use the centos7/8, vagrant >= 2.2.8 combo.

**Comment by oxr463** [comment]
@localhost8080 this is for Gentoo though.

**Comment by localhost8080** [comment]
@oxr463 sorry, I totally missed that;

I had a 'me too' on the sshfs not found on centos

**Comment by bngsudheer** [comment]
On CentOS 8, I had to run `dnf --enablerepo=PowerTools install fuse-sshfs`

**Comment by dustymabe** [comment]
This should be fixed in #111. There is a new release out. Please try it and let us know how it works.
